docs/faqs.md

```diff
@@ -4,18 +4,14 @@ title: Frequently Asked Questions
 sidebar_label: FAQs
 ---
 
-### Can I use my own custom domain or subdomain?
-
-Yes, please refer to the [documentation](./micros/custom_domains.md).
-
 ### What are all the different 'keys' for?
 
 Deta offers 3 types of secrets / keys / tokens:
 
-1. **Project Keys**: This type of key interacts with both your data and your resources inside a project (currently allowing reads and writes for the project's Bases). A project key will be created for you when you create a project, and you can manage all of a project's keys from the *'Settings'* option in any project's sidebar. If this key is leaked, the data stored in any of the connected project's Bases can be compromised. 
+1. **Project Keys**: This type of key interacts with both your data and your resources inside a project (currently allowing reads and writes for the project's Bases and Drives). A project key will be created for you when you create a project, and you can manage all of a project's keys from the *'Settings'* option in any project's sidebar. If this key is leaked, the data stored in any of the connected project's Bases and Drives can be compromised. 
 
 2. **API Keys**: This is an optional key to protect your Micro HTTP endpoint or to implement client-based rules. Creating API Keys from the Deta CLI is described [here](/docs/cli/commands#deta-auth-create-api-key). Read more about API Keys [here](https://en.wikipedia.org/wiki/Application_programming_interface_key). If this key is leaked, your micro endpoint can be used by unwanted clients.
 
-3. **Access Tokens**: This token lets other programs manage (create, modify & delete) your Micros, Projects and Bases on your behalf. It's mainly used to let you have programmatic access to our API. Use cases: CI/CD, Gitpod, GitHub Actions. Information on Deta Access Tokens can be found [here](/docs/cli/auth#deta-access-tokens). If this key is leaked, your whole Deta account is compromised.
+3. **Access Tokens**: This token lets other programs manage (create, modify & delete) your Micros, Projects, Bases and Drives on your behalf. It's mainly used to let you have programmatic access to our API. Use cases: CI/CD, Gitpod, GitHub Actions. Information on Deta Access Tokens can be found [here](/docs/cli/auth#deta-access-tokens). If this key is leaked, your whole Deta account is compromised.
```

Comment:
**Access Tokens**: This token lets you interact with Deta via the CLI from a remote environment. Use cases: CI/CD, Gitpod, GitHub Actions. Information on Deta Access Tokens can be found [here](/docs/cli/auth#deta-access-tokens). If this key is leaked, your whole Deta account is compromised.

^ or something like this? E.g. not mentioning explicitly they can hit any endpoint and use Deta as Infra-as-code?

Reply:
will update thanks